nailgun/entities.py

```diff
@@ -5117,16 +5137,14 @@ class Subscription(
 
     def __init__(self, server_config=None, **kwargs):
         self._fields = {
-            'activation_key': entity_fields.OneToOneField(ActivationKey),
-            'host': entity_fields.OneToOneField(Host),
+            'activation_key': entity_fields.OneToManyField(ActivationKey),
+            'name': entity_fields.StringField(),
             'organization': entity_fields.OneToOneField(Organization),
+            'provided_product': entity_fields.OneToManyField(Product),
             'quantity': entity_fields.IntegerField(),
-            'subscriptions': entity_fields.OneToManyField(Subscription),
+            'subscription': entity_fields.OneToOneField(Subscription),
+            'system': entity_fields.OneToManyField(Host),
         }
-        # Before Satellite 6.2 System entity was used instead of Host
```

Comment:
Not true, checked for both 6.2 and 6.3 - it's still 'system', not 'host'. And anyway master should not contain workarounds  for older sat versions as for now